fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/config/FineractProperties.java

```diff
@@ -295,4 +297,33 @@ public static class UserNotificationSystemProperties {
 
         private boolean enabled;
     }
+
+    @Getter
+    @Setter
+    public static class FineractLoanProperties {
+
+        private FineractTransactionProcessorProperties transactionprocessor;
```

Comment:
Java naming convention pls. Should be `transactionProcessor`, note the capital `P`